saleor/plugins/manager.py

```diff
@@ -383,6 +383,16 @@ def calculate_checkout_line_total(
             checkout_info.channel,
             discounts,
         )
+        # apply entire order discount
+        default_value = base_calculations.apply_checkout_discount_on_checkout_line(
```

Comment:
Is it expected? The `checkout_total` description says, that sale and voucher are applied to the amount.

Reply:
It doesn't include the `entire_order` voucher with `apply_once_per_order` set to `False`. I can update the descriptions of the `calculate_base_line_total_price` as it's not clear.

Reply:
I updated the docstring of `calculate_base_line_total_price`